src/parser/formatting-evaluator-module.ts

```diff
@@ -0,0 +1,108 @@
+import Decimal from "decimal.js";
+import { JSDOM } from "jsdom";
+import MarkdownIt from "markdown-it";
+import configuration from "../configuration/config-reader";
+import { CommentType, GetActivity } from "../get-activity";
+import { GithubComment, Module, Result } from "./processor";
+
+interface Multiplier {
+  formattingMultiplier: number;
+  wordValue: number;
+}
+
+export class FormattingEvaluatorModule implements Module {
+  private readonly _configuration: {
+    enabled: boolean;
+    multipliers: ({ type: (keyof typeof CommentType)[] } & Multiplier)[];
+    scores: { [key: string]: number };
+  } = configuration.formattingEvaluator;
```

Comment:
It's not clear from Github but I'm guessing `configuration` is of type `any`? It'd be better to make types for the whole configuration using Typebox so you can also verify it in runtime when configuration gets loaded

Reply:
That is correct. Completely agree, I just didn't want to dedicate time to this since I didn't even know if that would be the way we would handle the configuration. I will add basic checks to make sure the configuration is correct.